.github/docker_images/alpine-linux/entry.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Instead of a boolean can this take the path to the script to run, so in the future if we wanted to add other tests we could without needing to add more parameters to this script, e.g. Alpine + Address sanitizer.